src/assets/styles/_info.scss

```diff
@@ -0,0 +1,58 @@
+$block-class: 'info';
+@import 'colors';
+
+@at-root {
+    .#{$block-class} {
+      margin-left: 1rem;
+
+      &__icon-only {
+        display: block;
+
+        &--hide {
+          display: none;
+        }
+      }
+
+      &__info-button {
+        width: 1rem;
+        height: 1rem;
+        margin-top: 1.8rem !important; 
+        margin-left: 1rem !important;
```

Comment:
Bzzzt! Try not to use `!important`. We want to save that for emergencies.

If you're getting grief, see if you can increase specificity by making line 16 `& &__info-button` instead.